test/unit/verifier/lint/test_ansible_lint.py

```diff
@@ -0,0 +1,214 @@
+@pytest.fixture
+def _instance(patched_config_validate, config_instance):
+    return ansible_lint.AnsibleLint(config_instance)
+
+
+def test_config_private_member(_instance):
+    assert isinstance(_instance._config, config.Config)
+
+
+def test_default_options_property(_instance):
+    options = {
+        'default_exclude': [_instance._config.scenario.ephemeral_directory],
+        'exclude': [],
+        'x': [],
+    }
+
+    assert options == _instance.default_options
+
+
+@pytest.mark.parametrize(
+    'config_instance', ['_verifier_lint_section_data'], indirect=True)
+def test_name_property(_instance):
+    assert 'ansible-lint' == _instance.name
+
+
+def test_enabled_property(_instance):
+    assert _instance.enabled
+
+
+@pytest.mark.parametrize(
+    'config_instance', ['_verifier_lint_section_data'], indirect=True)
+def test_options_property(_instance):
+    x = {
+        'default_exclude': [_instance._config.scenario.ephemeral_directory],
+        'exclude': [
+            'foo',
+            'bar',
+        ],
+        'x': [
+            'foo',
+            'bar',
+        ],
+        'foo': 'bar',
+        'v': True,
+    }
+
+    assert x == _instance.options
+
+
+@pytest.mark.parametrize(
+    'config_instance', ['_verifier_lint_section_data'], indirect=True)
+def test_options_property_handles_cli_args(_instance):
+    _instance._config.args = {'debug': True}
+    x = {
+        'default_exclude': [_instance._config.scenario.ephemeral_directory],
+        'exclude': [
+            'foo',
+            'bar',
+        ],
+        'x': [
+            'foo',
+            'bar',
+        ],
+        'foo': 'bar',
+        'v': True,
+    }
+
+    assert x == _instance.options
+
+
+def test_default_env_property(_instance):
+    assert 'MOLECULE_FILE' in _instance.default_env
+    assert 'MOLECULE_INVENTORY_FILE' in _instance.default_env
+    assert 'MOLECULE_SCENARIO_DIRECTORY' in _instance.default_env
+    assert 'MOLECULE_INSTANCE_CONFIG' in _instance.default_env
+
+
+@pytest.mark.parametrize(
+    'config_instance', ['_verifier_lint_section_data'], indirect=True)
+def test_env_property(_instance):
+    assert 'bar' == _instance.env['FOO']
+    assert 'ANSIBLE_CONFIG' in _instance.env
+    assert 'ANSIBLE_ROLES_PATH' in _instance.env
+    assert 'ANSIBLE_LIBRARY' in _instance.env
+    assert 'ANSIBLE_FILTER_PLUGINS' in _instance.env
+
+
+@pytest.mark.parametrize(
+    'config_instance', ['_verifier_lint_section_data'], indirect=True)
+def test_bake(_instance, monkeypatch):
+    monkeypatch.setattr(os.path, 'exists', lambda x: True)
+    _instance.bake()
+    x = [
+        str(sh.ansible_lint),
+        '--foo=bar',
+        '-v',
+        '-x',
+        '-x',
+        '--exclude={}'.format(_instance._config.scenario.ephemeral_directory),
+        '--exclude=foo',
+        '--exclude=bar',
+        _instance._playbook,
+        'bar',
+        'foo',
+    ]
+    result = str(_instance._ansible_lint_command).split()
```

Comment:
Isn't it a string already?
```suggestion
    result = _instance._ansible_lint_command.split()
```

Reply:
No, it's an `sh.Command` object.